src/client/components/Navigation/NavAssessment/Header/Title/CycleSwitcher.tsx

```diff
@@ -17,10 +18,11 @@ const CycleSwitcher = () => {
   const assessment = useAssessment()
   const user = useUser()
   const isDataLocked = useIsDataLocked()
+  const isDataExport = countryIso && !Areas.isISOCountry(countryIso)
```

Comment:
This condition is the same used in pages/Assessment.tax

I also noticed there's a hook useIsDataExportView that should be used in both cases .
Can u update those?

Reply:
I renamed the variables to isDataExportView as well.